.github/workflows/main.yml

```diff
@@ -140,6 +131,11 @@ jobs:
           docker cp $GITHUB_WORKSPACE/l10n singleton:/
           cd $GITHUB_WORKSPACE
           cp -r $GITHUB_WORKSPACE/devops/autotest/client/ruby/l10n/bundles/* $GITHUB_WORKSPACE/l10n/bundles
+          cd $GITHUB_WORKSPACE
+          sed -i 's|singleton-ruby|singleton-client|g' singleton-ruby.gemspec
+          mv singleton-ruby.gemspec singleton-client.gemspec
+          gem build singleton-client.gemspec
```

Comment:
These lines should be done in client code.

Reply:
These lines are naming mapping for ruby client and smoke test, how about keep it here so both ruby client and smoke test don't need do anything.

Reply:
This is an issue of client and should be fixed.

Reply:
Please merge lastest change from upstream branch.